spec/features/courses/new_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
```

Comment:
shouldn't this file sit with 

spec/features/courses/vacancies/edit_spec.rb

instead of 

spec/features/courses/new_spec.rb

Reply:
@defong This feature spec is for the course creation overall ... these steps should be tested as whole journey through the create course, and we'll probably have multiple feature specs for course creation to test out different scenarios.

The request spec exercises this functionality in isolation.